integration/terraform/ec2/linux/README.md

```diff
@@ -19,3 +19,64 @@ AMI builder pipeline builds the ami
 The pipeline installs required packages and updates ami software
 
 This process generates a new ami we can then use for testing
+
+**How to integration test in your aws account**
+1. Create resources and setup local
+   1. Install terraform
+   2. Set up aws credentials
+   3. Set up iam role
+      1. Role must include cloud watch server policy and s3 policy
+   4. Create s3 bucket
+2. Upload binary to s3
+   1. make release 
+      1. hints
+         1. You may want to do this on a linux ec2 instance installing the agent may fail if you build on Mac 
+         2. If you want to build faster remove not needed packages and test from release step
```

Comment:
Could change this to recommend users run:
`make build && make package-rpm` or `package-deb` (depending on which OS type will be tested).